grammars/repositories/flavors/rmarkdown.cson

```diff
@@ -5,27 +5,20 @@
 key: 'rmarkdown'
 patterns: [
   {
-    begin: '(?x)
-      (```)(
-        (\\{)(r)(?:\\s)?
-        (?:([a-zA-Z0-9|_|-]*)(?=[\\s|,|}])\\s?)?
-        ([^\\}]*)?
-        (\\})
-      )
-    '
-    end: '^(\\1)$'
+    begin: '(?:\\n|^)(`{3,})[ \\.]*(\\{)(\\S*) *(?:([^= ]*) *(?=[,\\}]))? *,?(.*?)(\\}) *(?=\\n|$)'
```

Comment:
I can't, because the regex contains actual white space. I don't know how to split it to multi lines in this case.